datetime_tz/win32tz_map.py

```diff
@@ -0,0 +1,110 @@
+'''Map between Windows and Olson timezones taken from http://www.unicode.org/repos/cldr/trunk/common/supplemental/windowsZones.xml
```

Comment:
Nope, no reason, but obviously we still want to do that at build rather than run time, so I need to do it in a way that the build still succeeds when the file's not there yet